portfolio/src/main/webapp/StyleMenu.css

```diff
@@ -0,0 +1,94 @@
+body{
+    background: whitesmoke;
+
+}
+#socials {
+  margin:0;
+  color: #dbd8d8;
+}
+.projects {
+  background: #a8a7a75e;
+  background-size: cover;
+  height: 600px;
+  position: relative;
+}
+#projects{
+    border: none;
+}
+.experience {
+    background: rgba(117, 117, 116, 0.623);
+  background-size: cover;
+  height: 600px;
+  position: relative;
+}
+#experience{
```

Comment:
I would put all your classes together and your ids together. for example all the # should be together and all the . should be together. Also I would probably try to have some sort of sorting pattern like doing these in alphabetical order because when you start getting to a larger website, this will get very unwieldy and large pretty quickly.